app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return "Command must be exactly 3 characters"
+    command, initial_file, final_file = command.split()
```

Comment:
Do not perform `split` twice, perform it once instead
```suggestion
    command_split = command.split()
    if len(command_split) != 3:
        return "Command must be exactly 3 characters"
    command_name, initial_file, final_file = command_split
```

Reply:
This comment is not implemented, do not perform `split` twice, perform it once instead as mentioned [here](https://github.com/mate-academy/py-copy-file/pull/877/files#r1532498104)